### DISCUSSION TITLE
Chat hover action

### DISCUSSION THREAD

**Discussion by fizzrepo** [Zrips/CMI#7192] [Enhancement]
I am using the CMI chat function for my server, and I thought it'd be cool if you could have actions for clicking on certain components of the chat message, for example clicking the prefix, in this case being **OWNER**.
![image](https://user-images.githubusercontent.com/58730469/175102433-24177b63-0060-4e84-9eb9-55264280946f.png)
 it'd show a hover menu with stats about the player and when you click it, it shows a command menu which you can customise. By default it'd show options such as Ignore, Message and Afkcheck, looking something like this.
![image](https://user-images.githubusercontent.com/58730469/175103296-46664dda-21a7-4957-9c54-9ca19ebd7586.png)
The hover menu would be able to display placeholders and stuff.

Quite a complex suggestion, but I think it'd be an amazing addition to CMI.

**Comment by mrfloris** [comment]
Not your solution, but one.. you can set 1 command to run on click as far as I know, you could set that command to load another command that does more .. or like a custom GUI menu from another plugin. 

So they see: (for example)
```
<bob> hi everybody
```
and you click on the chat msg, you get: (for example) to run the command of a cmi aliaseditor made /_chatCommands command, that shows:
```
<bob> hi everybody
[bob] -> options: kick, warn, jail, ban, report
[bob] -> options: msg, invite chatroom, tag
[bob] -> show info, check account, give 5k
```
And those are then `cmi ctellraw [playerName] <T>..</T> made lines that you can click on to run those commands. 
or like , it runs `bossshoppro open chatMenu` with GUI buttons you can click on.. 

so technically it's possible, but via via. and not 100% how you want, would be fun to have clickable actions for staff in front of the chat, yep.
